airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/bump_version/pipeline.py

```diff
@@ -85,9 +84,9 @@ def add_changelog_entry(self, og_doc_content: str) -> str:
         today = datetime.date.today().strftime("%Y-%m-%d")
         lines = og_doc_content.splitlines()
         line_index_for_new_entry = self.find_line_index_for_new_entry(og_doc_content)
-        new_entry = f"| {self.new_version} | {today} | [{self.pull_request_number}](https://github.com/airbytehq/airbyte/pull/{self.pull_request_number}) | {self.changelog_entry} |"
+        new_entry = f"| {self.new_version:8} | {today} | [{self.pull_request_number}](https://github.com/airbytehq/airbyte/pull/{self.pull_request_number}) | {self.changelog_entry} |"
```

Comment:
I don't think I get the effect of `self.new_version:8`.
The syntax to select of substring is `self.new_version[:8]`.
If the intent is to cut the string it also don't follow why.
`self.new_version` can be of any length right? Is it problematic if it's longer or shorter than 8 characters?
You mentionned:
>  increasing the space for the version (so we can accomodate 0.21.53, or 0.125.2)

But the space is not fixed so  0.21.53, or 0.125.2 are likely already rendered correctly in the changelog.

Am I missing something?

Reply:
the goal was simply to avoid 
```
1.10.0 |
1.9.0 |
1.8.0|
```
and instead to have
```
1.10.0  |
1.9.0   |
1.8.0   |
```
I can remove it

Reply:
I'm ok with the goal but I don't think  `self.new_version[:8]` or `self.new_version:8` has the desired effect. I think `self.new_version.ljust(8, " ")` is more correct.